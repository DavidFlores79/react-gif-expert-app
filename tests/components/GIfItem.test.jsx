const { render } = require("@testing-library/react")
const { GIfItem } = require("../../src/components/GIfItem")

describe('Pruebas en Gif item', () => { 
    
    test('debe ser igual que el snapshot', () => { 

        const title = "GOT";
        const url = 'https:got.com/imagen.png'

        const { container } = render( <GIfItem title={ title} url={url}/> )

        expect( container ).toMatchSnapshot();

     })
 })