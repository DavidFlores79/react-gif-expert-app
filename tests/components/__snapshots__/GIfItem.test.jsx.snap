// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en Gif item debe ser igual que el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="GOT"
      src="https:got.com/imagen.png"
    />
    <p>
      GOT
    </p>
  </div>
</div>
`;
